src/appforms/src/backbone/040-view06Steps.js

```diff
@@ -71,12 +71,14 @@ StepsView = Backbone.View.extend({
 
     self.$el.find('li:eq(' + displayIndex + ')').addClass('active');
 
-    if(pageName.length === 0){
-      pageName = "Page " + (displayIndex + 1);
+    var hasCustomPageName = pageName.length !== 0;
```

Comment:
I think this change is only hiding the page title. Am I understanding it correctly.

I think the idea is to hide the big page number at the top of the form if there is only a single page.